### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewSearch_ScreenReader: VoiceOver does not read the necessary headings on the screen.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52448]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : HighSierra10.13.5

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader : VoiceOver

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Search" button.
3. Navigate to "Search" section and read the content "Search" using basic navigation. Read the heading using Ctrl + Option + Command  + H.

**Actual:**
Screenreader does not read the content "Search" as appropriate heading which is so.

**Expected:**
Screenreader should read- "Search" as heading with appropriate heading level.

**Recommendations:**
Add appropriate native heading level tag to the content "Search".
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the appropriate headings present on the screen.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[A11y_VSCode_HeadingsIssue.pptx](https://github.com/Microsoft/vscode/files/2118947/A11y_VSCode_HeadingsIssue.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
I don't know what it considers a 'heading', I think this is a general workbench issue.

**Comment by bpasero** [comment]
Everyone is part of the workbench ❤️

**Comment by roblourens** [comment]
Haha true. What I really mean is, this is one of a a few issues where the testers opened the exact same issue on every part of the workbench, so we need to apply the same solution everywhere and not solve it independently for each piece.
